source/solvers/vof_assemblers.cc

```diff
@@ -107,13 +104,11 @@ VOFAssemblerCore<dim>::assemble_matrix(VOFScratchData<dim> &      scratch_data,
 
           // Strong Jacobian associated with the GLS
           // stabilization
-          strong_jacobian_vec[q][j] += velocity * grad_phi_phase_j +
-                                       phi_phase_j * velocity_divergence -
-                                       diffusivity * laplacian_phi_phase_j;
+          strong_jacobian_vec[q][j] +=
+            velocity * grad_phi_phase_j - diffusivity * laplacian_phi_phase_j;
 
-          // DCDD shock capturing
-          if (DCDD)
-            strong_jacobian_vec[q][j] += -vdcdd * laplacian_phi_phase_j;
```

Comment:
Is this removed in order to add diffusivity only in the right direction, or was it a previous error to add it?

Reply:
Yeah it was a bad idea to add it actually